build.gradle

```diff
@@ -22,13 +22,20 @@ ext {
     // Most recent minor versions of all major releases
     // between earliest supported (incl.) and latest stable (excl.), used for binary compatibility checks
     latestMinorsOfOldSupportedMajors: ['2020.1.4', '2020.2.4'],
-    latestSupportedMajor: '2020.3',
```

Comment:
One less value to bump manually ;p
This value (corresponding to `until-build`) can be derived automatically, see below